docker/nwm_gui/docker-compose.yml

```diff
@@ -52,7 +52,9 @@ services:
       - SQL_ENGINE=django.db.backends.postgresql
       - SQL_DATABASE=${DMOD_GUI_POSTGRES_DB:-dmod_dev}
       - SQL_USER=${DMOD_GUI_POSTGRES_USER:?}
-      - SQL_HOST=nwm_gui_db
+      #- SQL_HOST=nwm_gui_db
```

Comment:
These commented sections should probably just be removed since we know they don't work.